edb/server/args.py

```diff
@@ -313,6 +316,14 @@ def _writer(status: str) -> None:
         '--auto-shutdown', type=bool, default=False, is_flag=True,
         help='shutdown the server after the last ' +
              'connection is closed'),
+    click.option(
+        '--ha-mode', default='off',
```

Comment:
I wonder if we should use the dsn-like URI format for this instead of separate options, e.g:

`--ha-cluser=stolon+consul://<consul-agent-addr>:8500/<cluster-name>`

This ways we'd also be able to configure Consul connection parameters if needed (e.g. auth, TLS etc).

Reply:
+1